ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+clang-format -style=file -i stl/inc/** stl/src/** 2>&1

```

Comment:
This misses the subdirectories, no? `**` doesn't do anything special in CMD to my knowledge.

Reply:
The ** is getting processed by clang-format, not by cmd. (I'm inferring that subdirectories are getting checked by the number of "is a directory" outputs we're getting)

Reply:
`find stl/inc stl/src -type f | xargs clang-format --style=file -i` works in bash ;)